MVP/MVPUI/ContributionsPage.xaml.cs

```diff
@@ -82,6 +82,32 @@ public ContributionsPage()
             }
         }
 
+        private bool _allContributionsLoaded = false;
+        public async void OnItemAppearing(object sender, ItemVisibilityEventArgs eventArgs)
+        {
+            if (!_allContributionsLoaded && !ListViewContributions.IsRefreshing)
```

Comment:
The view mode has a "CanLoadMore" property we should use.

if the returned result of the first query of 50 is < 50 then we can load more

We should also use the IsBusy flag here